### PR TITLE
Increase minimum encryption key size

### PR DESCRIPTION
**Pull request by huguesBouvier**
Increase minimum encryption key size
Description
-----------
Increase minimum encryption key size
Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ x] My code is Linted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gatt_server.c [hunk 1]
```diff
@@ -888,6 +888,7 @@ BTStatus_t prvAddServiceBlob( uint8_t ucServerIf,
                     pCharacteristics[ charCount ].arg = ( void * ) pxService;
                     pCharacteristics[ charCount ].access_cb = prvGATTCharAccessCb;
                     pCharacteristics[ charCount ].flags = prvAFRToESPCharPerm( pxService->pxBLEAttributes[ index ].xCharacteristic.xProperties, pxService->pxBLEAttributes[ index ].xCharacteristic.xPermissions );
+                    pCharacteristics[ charCount ].min_key_size = btKEY_MAX_LEN;
```
Comment (qiutongs) [inline vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gatt_server.c]: Is it a typo? It should be set to 'btKEY_MIN_LEN'?

File: vendors/nordic/boards/nrf52840-dk/ports/ble/iot_ble_hal_common_gap.c [hunk 2]
```diff
@@ -158,8 +158,8 @@ ble_gap_sec_params_t xSecurityParameters =
     .keypress       = 0,
     .io_caps        = BLE_GAP_IO_CAPS_DISPLAY_YESNO,
     .oob            = 0,
-    .min_key_size   = 7,
-    .max_key_size   = 16,
+    .min_key_size   = btKEY_MAX_LEN,
```
Comment (qiutongs) [inline vendors/nordic/boards/nrf52840-dk/ports/ble/iot_ble_hal_common_gap.c]: Is it a typo? It should be set to 'btKEY_MIN_LEN'?

Comment (huguesBouvier) [reply vendors/nordic/boards/nrf52840-dk/ports/ble/iot_ble_hal_common_gap.c]: It is not :), we increase the min size to max. But I changed it anyway for less confusion

File: vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gatt_server.c [hunk 3]
```diff
@@ -38,6 +38,7 @@
 #include "bt_hal_manager.h"
 #include "bt_hal_gatt_server.h"
 #include "iot_ble_hal_internals.h"
+#include "iot_ble_config.h"
```
Comment (ravibhagavandas) [inline vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gatt_server.c]: Config file include before other includes?

Comment (huguesBouvier) [reply vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gatt_server.c]: It is fine, the other includes are low level. But for consistency sake, I ll move it up

**Pull request merged**